etcd_test.go

```diff
@@ -28,3 +28,24 @@ func Test_Storage(t *testing.T) {
 
 	storagetest.Test(t, storage)
 }
+
+func Test_Storage_Prefix(t *testing.T) {
+	etcdConfig := clientv3.Config{
+		Endpoints: []string{"http://127.0.0.1:2379"},
+	}
+	etcdClient, err := clientv3.New(etcdConfig)
+	if err != nil {
+		t.Fatal("expected", nil, "got", err)
+	}
+
+	config := DefaultConfig()
+	config.EtcdClient = etcdClient
+	config.Prefix = "test-etcd-prefix"
```

Comment:
What happens when the prefix contains slashes? I think it would be good to have a test for this as well.